volttrontesting/fixtures/docker_wrapper.py

```diff
@@ -47,29 +51,44 @@ def create_container(image_name: str, ports: dict = None, env: dict = None, comm
 
         # Create docker client (Uses localhost as agent connection.
         client = docker.from_env()
-        if ":" in image_name:
-            client.images.pull(image_name)
-        else:
-            # So all tags aren't pulled. According to docs https://docker-py.readthedocs.io/en/stable/images.html.
-            client.images.pull(image_name + ":latest")
-        container = client.containers.run(image_name, ports=ports, environment=env, auto_remove=True, detach=True)
+
+        try:
+            repo = image_name
+            if ":" not in repo:
+                # So all tags aren't pulled. According to docs https://docker-py.readthedocs.io/en/stable/images.html.
+                repo = repo + ":latest"
+            client.images.pull(repo)
+        except APIError as e:
+            raise RuntimeError(e)
+
+        try:
+            container = client.containers.run(image_name, ports=ports, environment=env, auto_remove=True, detach=True)
+        except ImageNotFound as e:
```

Comment:
I think you could do 

````
except (ImageNotFound, RuntimeError) as e:
````

Reply:
Yes, you're right. ImageNotFound is a NotFound, which is also an APIError. Will change.